client/public/index.html

```diff
@@ -10,7 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
```

Comment:
not a fan of PT Sans? maybe we can get access to our typekit account and use Proxima nova as used on the battlesnake.io site

Reply:
Lato is what's used for the main site so I figured I'd just change it here. If you had settled on PT Sans, by all means use it, I was just going for conformance and consistency. I also think I didn't change any other `font-family` declarations so this would actually break things before it did anything good :)

Reply:
Actually, I went through and made changes to the `font-family` declarations and now it's using Lato. FIGHT ME! 💪 😝